src/platforms/common/data-management/debug-files.mdx

```diff
@@ -54,7 +59,9 @@ Sentry differentiates in four kinds of debug information:
   to as "stack unwinding" or "stack walking". Since this is also required when
   throwing exceptions in C++, this information is often included in the
   executable or library. If an uploaded file contains this information, it shows
-  the `unwind` tag.
+  the `unwind` tag.  Note that on some platforms no actual unwinding takes place.
+  For instance, WebAssembly currently does not have the equivalent of minidumps
+  which means we do not require that type of information in such cases.
```

Comment:
I'm not sure if it is required to call WASM out here. The same goes for all platforms where we provide stack traces from the client: Cocoa, Android, as well as Linux with the in-process backend. The first sentence therefore reads "from Minidumps", which simply doesn't apply.

Can we just hide unwind information from this page alltogether?